13/alpine3.20/Dockerfile

```
@@ -4,6 +4,14 @@
 # PLEASE DO NOT EDIT IT DIRECTLY.
 #
 
+FROM golang:1.23.6 AS gosubuilder
+ENV GOSU_VERSION=1.17
```

Please note that ENV variables are propagated to runtime, I suggest using ARG variables during build.
In addition, there have been 33 commits since 1.17, rebuilds could actually use the latest code instead of an outdated tag.
```suggestion
ARG GOSU_VERSION=1.17
```